src/game/game.hpp

```diff
@@ -0,0 +1,16 @@
+#ifndef GAME_HPP
+#define GAME_HPP
+
+#include <chrono>
+
+class Game {
```

Comment:
Should we wrap this in some namespace? `semseterser` for example?

Reply:
Yes, good idea!